.jshintrc

```diff
@@ -0,0 +1,30 @@
+{
+  "nonew": true,
+  "jquery": false,
+  "curly": true,
+  "latedef": true,
+  "unused": true,
+  "noarg": true,
+  "indent": 1,
+  "trailing": true,
+  "white": true,
+  "forin": true,
+  "noempty": true,
+  "quotmark": "single",
+  "eqeqeq": true,
+  "strict": true,
+  "undef": true,
+  "bitwise": true,
```

Comment:
the bitwise is accepted for rdm.js and solved all of the errors pretty much, do we need to allow it globally (i.e. everywhere)?